patches/server/0682-Missing-Entity-Behavior-API.patch

```diff
@@ -287,3 +393,113 @@ index 2d2620dbb16aec850e8afda02174508a4be5a313..ba4e6deaaa725296be830324d2c64868
  
      public static Gene fromNms(net.minecraft.world.entity.animal.Panda.Gene gene) {
          Preconditions.checkArgument(gene != null, "Gene may not be null");
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPolarBear.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPolarBear.java
+index da1488c9cae53bd554727c850da2192adda2478a..30a0eac179c86b0fe94a2a40b5bfcd3eee01e53b 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPolarBear.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPolarBear.java
+@@ -23,4 +23,16 @@ public class CraftPolarBear extends CraftAnimals implements PolarBear {
+     public EntityType getType() {
+         return EntityType.POLAR_BEAR;
+     }
++
++    // Paper start
++    @Override
++    public boolean isStanding() {
++        return this.getHandle().isStanding();
++    }
++
++    @Override
++    public void setStanding(boolean standing) {
++        this.getHandle().setStanding(standing);
++    }
++    // Paper end
+ }
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftRaider.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftRaider.java
+index 27a8a2b1e03254b1fc6fe8edc3ff77841a42f5f6..8d77b870fd9de69b57ae1affdfbd2a02f62e75c7 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftRaider.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftRaider.java
+@@ -58,4 +58,16 @@ public abstract class CraftRaider extends CraftMonster implements Raider {
+     public void setCanJoinRaid(boolean join) {
+         this.getHandle().setCanJoinRaid(join);
+     }
++
++    // Paper start
++    @Override
++    public boolean isCelebrating() {
++        return this.getHandle().isCelebrating();
++    }
++
++    @Override
++    public void setCelebrating(boolean celebrating) {
++        this.getHandle().setCelebrating(celebrating);
++    }
++    // Paper end
+ }
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftTrident.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftTrident.java
+index bf5b2fd6676c4430578db4cc6c603c501cc5e349..832981b07ef5c633ef00a382f56798ee87eec0df 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftTrident.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftTrident.java
+@@ -37,4 +37,27 @@ public class CraftTrident extends CraftArrow implements Trident {
+     public EntityType getType() {
+         return EntityType.TRIDENT;
+     }
++
++    // Paper start
++    @Override
++    public boolean hasGlint() {
```

Comment:
both glint and foil are kinda strange names, why not glowing?

Reply:
Fun things, https://papermc.io/javadocs/paper/1.18/org/bukkit/entity/Entity.html#isGlowing()

Reply:
Yea glowing and if it has a glint or not isn't the same thing. I think glint is fine, as it even matches with this PR for itemstack glint (if we are still waiting on mojang for this one as well) 

https://github.com/PaperMC/Paper/pull/5725/files